data-showcase-api-e2e/src/test/groovy/config/Config.groovy

```diff
@@ -39,4 +39,6 @@ class Config {
     //settings
     public static final boolean AUTH_NEEDED = false
     public static final String DEFAULT_USER = 'default'
+    public static final String TOKEN = 'zh6WWMJcnIbRqU4KnK0sFPRapldWK6PS7hC5Gnm/GYU='
```

Comment:
I would expect the application to run in test environment (either `-Dgrails.env=test` or `-Dgrails.env=testInternal`). Then the token should be `TestToken123!`, right?